CODE_REVIEW.md

```diff
@@ -0,0 +1,65 @@
+Your PR needs to pass all the reviews steps as fast as possible, you have to communicate and urge the reviewers everytime the PR needs to be reviewed.
+You are invited to read carefully your PR before request a review, this can drastically reduce the time your PR stays open before being approved.
+Reading multiple times your PR changes in diff tools like github can help you spot parts that can be improved.
+
+## PR steps
+### Open a PR
+
+Be sure you are providing the necessary information to speed-up the PR process an to help reviewers doing their part:
+
+* Title should contain the reference to the story/ticket if existing
+* Title should summarise the purpose of the implementation by the code POV, not replicate the title of the story that represent usually the user POV
+* Description
+  1. Should contain link to useful references or resources if available (Jira ticket or tickets, libraries, external documentation...)
+  2. You should provide the right context and background information to help reviewers undestand your implemntation choices, avoided un-necessary subsequent questions ('I used xxx because of xyz...')
+  3. Tests plan: You should provide a basic tests plan. This is to communicate to reviewers that you knew how the feature is expected to work and can also help human QA.
+
+### Assign reviewers
+
+PRs need to be **'peer approved'** and **'core approved'**.
+A peer reviewer is a developer with (very approssimatively) a similar amount of skills and the same seniority and knowledge of the product.
+A core reviewer is a developer with more Seniority and a bigger knowledge of the product and of the 'big picture', usually with also wider background of the business requirements.
+
+Once you are ready, you should assign a **peer reviewer** and contact him/her directly or mentioning in the appropriate PR channel.
+If he/she is busy he/she can ask you to find some else available. 
+On the meantime you should also label the PR **'Waiting for review'**.
+
+When your **peer reviewer** is happy, he/she have to put the label **'Peer approved'** and you can ask a core reviewer to proceed with his/her review.
+
+When the **core reviewer** is happy, he/she have to put the label **'Ready for QA'** and assign the PR to one or more QA.
+The **core reviewer** have to communicate in the your project channel that the story is RFQA (Ready for QA). This can easily done on Slack with the 'jirio' command like: ```/jirio transit WP-1234 Ready for QA```.
+
+This will automatically update the ticket in JIRA accordingly, anyway you are responsible to keep your tickets in the Kanban board always up-to-date.
+
+A **core review** is inclusive of a **peer review**. This mean that if a **core reviewer**'s PR is approved by another **core reviewer** is automatically RFQA.
+
+Anyway, core reviewers are invited to ask the review also of a non-core reviewer to spread knwoledge and give them an additional opportunity to learn.
+
+### Labelling system
```

Comment:
A few of these feel redundant as we can probably take better advantage of what you can do in Github for things like request changes and comments.  I'd roughly recommend:
-Use Github's "Reviewers" tool to assign reviewers
-Reviewers use the "Review Changes" functionality in Github and the Approve/Comment/Request Changes options
-Enable Github's Required Reviews so that all PRs require at least one review to me merged (admin can override this)
-Use labels for: Managing number of reviews, Draft, Do Not Merge, Build PR and maybe some team internal stuff. 

Generally I think labels are better served for team level communication and we should use more of what Github provides for the review process so that is kept standard across teams.

Reply:
👍 Will implement you suggestions